### DISCUSSION TITLE
non-varying axis labels and bounds in annotations

### DISCUSSION THREAD

**Discussion by jfavre** [Alpine-DAV/ascent#940] [Q&A] [answered]
I have an SPH code instrumented with Ascent. Particles move around and their global bounding box varies in time. I would like to use scenes["s1/renders/r1/annotations"] = "true", but I can't find documentation on how to hard-code the bounds of the XYZ axis in annotations to avoid the ugly jittering of the axis labels. Can it be done at all?
![image 00001](https://user-images.githubusercontent.com/7058064/171336520-fbb98b4b-fc48-4313-8c4e-d8449ee42985.png)
![image 00100](https://user-images.githubusercontent.com/7058064/171336526-7f5cbd63-276b-47c4-8304-0402a54562e1.png)

**Comment by cyrush** [comment]
@jfavre  Looked into this, currently:

- We can't override the spatial bounds of the  dataset to specify a different bounding box
- VTK-m does not expose options to control the number of ticks or where they are placed (which could help mitigate somewhat)

I think adding an option to set the spatial bounds is a useful one. Since it normally comes from the dataset, we need to think about the implications.  I'll add a ticket for this.

**Reply by cyrush** [reply]
Added related issue ticket: https://github.com/Alpine-DAV/ascent/issues/942

**Comment by nicolemarsaglia** [comment, marked as answer]
An ability to overwrite the rendered boundaries is being added in #968 
This should help in stabilizing the annotations of your images.
